src/Catty/DataModel/VariablesContainer/VariablesContainer.h

```diff
@@ -47,6 +47,9 @@
 - (NSArray*)allVariablesForObject:(SpriteObject*)spriteObject;
 
 // Array of UserVariable
+- (NSArray*)allVariables;
+
+// Array of UserVariable
 - (NSArray*)objectVariablesForObject:(SpriteObject*)spriteObject;
```

Comment:
I think you do not need this member (+concurrent methods) anymore. :ear_of_rice: